app/src/main/java/com/portfolio/tasky/di/modules/ValidatorsModule.kt

```diff
@@ -0,0 +1,24 @@
+package com.portfolio.tasky.di.modules
+
+import com.portfolio.tasky.usecases.*
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+class ValidatorsModule {
+        @Provides
+        @Singleton
+        fun provideEmailPatternValidationImpl(): EmailPatternValidator {
```

Comment:
Try not to add "Impl" to method names either
So in this case it would be 

"provideEmailPatternValidation"

Reply:
Okay

Done